<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NUMERICAL_METHODS</class>
 <widget class="QMainWindow" name="NUMERICAL_METHODS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PROYECTO</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Seleccione un Método&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SALIR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Salir</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>136</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="GAUSS_SEID">
       <property name="text">
        <string>Gauss Seidel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NEWTON">
       <property name="text">
        <string>Newton Raphson</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ELIMINACION_GAUSS">
       <property name="text">
        <string>Eliminacion Gausiana</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PUNTO_FIJO">
       <property name="text">
        <string>Iteración de Punto Fijo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LAGRANGE">
       <property name="text">
        <string>Interpolación de Lagrange</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>133</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="REGRELINEAL">
             <property name="text">
              <string>Regresion Lineal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="REGMULT">
             <property name="text">
              <string>Regresion Lineal Multiple</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DERIVACION">
             <property name="text">
              <string>Derivacion</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="IntegracionNumerica">
           <property name="text">
            <string>Integración numérica</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="MetodoDeEuler">
         <property name="text">
          <string>Método de Euler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="MetodoDeRungeKutta">
       <property name="text">
        <string>Método de Runge Kutta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
